core/schemas/model.py

```diff
@@ -21,6 +17,43 @@ def id(self):
         return self.__id
 
 
+class YetiAclModel(YetiBaseModel):
+    _acls: dict[str, RoleRelationship] = {}
+
+    def __init__(self, **data):
+        super().__init__(**data)
+        for name, value in data.get("acls", {}).items():
+            self._acls[name] = RoleRelationship(**value)
+
+    @computed_field(return_type=dict[str, RoleRelationship])
+    @property
+    def acls(self):
+        return self._acls
+
+    def get_acls(self) -> None:
```

Comment:
What's the difference between `_acls` in __init__ and the one updated here? and by extension, should I use `acls` or `get_acls`?

Reply:
The assignments in `__init__` are only for when the query returns ACLs as part of the document that is read from the database (such as when specifying ACLs in the graph query)

There are cases where ACLs aren't loaded from the db (e.g. when calling .get or .find).

`.get_acls` works the same way as `.get_tags`, in the sense that it will do that graph query for you and return an object with populated acls.